style.css

```diff
@@ -0,0 +1,62 @@
+h1,h2,p{
+    font-family: sans-serif;
+}
+
+img{
+    width: 600px; 
+    height: 250px;
+}
+h2{
+    margin-top: 0;
```

Comment:
Instead of this line, you can use:
```css
* {
margin:0;
padding:0;
}
```
for a fresh clean start with no margin or padding, since most HTML elements have default margin and padding.

Reply:
Right, but this would also strip off the bottom margin of h2 and margin and padding of other html elements, which I don't want. The aim of this rule is to pull the h2 a bit up because I felt it is kind of off with top margin.

Reply:
Yeah, both ways work. But in my opinion this way gives you more control on the page style. So you can give each element on the page your own style, not the default one 😅

Reply:
Thanks again. You've been so helpful.